package.json

```diff
@@ -38,24 +38,23 @@
     "date-fns-tz": "^2.0.0",
     "dom-to-image": "^2.6.0",
     "encoding": "^0.1.13",
-    "eslint": "8.47.0",
-    "eslint-config-next": "13.4.19",
```

Comment:
Moved to dev dependencies, this is the reason for `punycode` as a non-dev dependency.